habitat-lab/habitat/tasks/rearrange/rearrange_grasp_manager.py

```diff
@@ -67,7 +72,9 @@ def is_violating_hold_constraint(self) -> bool:
         Returns true if the object is too far away from the gripper, meaning
         the agent violated the hold constraint.
         """
-        ee_pos = self._managed_robot.ee_transform.translation
+        ee_pos = self._managed_articulated_agent.ee_transform(
+            self.ee_index
+        ).translation
```

Comment:
Is this more compact and more readable ?
```suggestion
        ee_transform = self._managed_articulated_agent.ee_transform(self.ee_index)
        ee_pos = ee_transform.translation
```

Reply:
I agree! Somehow `black` modified this code.